src/styles/_catalog.scss

```
@@ -1,22 +1,27 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

```suggestion
  grid-template-columns: repeat(1, 200px);
```

При видаленні ось цього особисто моя реалізація виглядала так само, як і з цим правилом, тому не знаю чи воно необхідне. Можу дуже грубо помилятись, просто помітив :)

---

погоджуюсь, repeat тут не потрібен